internal/wrexec/recording_cmd.go

```diff
@@ -147,3 +147,8 @@ func (rf *RecordingCommandFactory) Wrap(ctx context.Context, logger log.Logger,
 	store := rcache.NewFIFOList(KeyPrefix, rf.maxSize)
 	return RecordingWrap(ctx, logger, rf.shouldRecord, store, cmd)
 }
+
+// NewNoOpRecordingCOmmandFactory is a recording command factory that is intiailised with a nil shouldRecord and maxSize 0. This is a helper for use in tests.
+func NewNoOpRecordingCOmmandFactory() *RecordingCommandFactory {
```

Comment:
typos fix (will also need to update call sites):
```suggestion
// NewNoOpRecordingCommandFactory is a recording command factory that is initialised with a nil shouldRecord and maxSize 0. This is a helper for use in tests.
func NewNoOpRecordingCommandFactory() *RecordingCommandFactory {
```

Reply:
Fixed.